src/test/java/com/rocana/lucene/codec/v1/TestRocanaSearchCodecV1.java

```diff
@@ -113,31 +106,14 @@ public void getActualPostingsFormatShouldReturnPostingsFormatFromSPI() {
   }
 
   /**
-   * This test ensures we don't rename the codec class and forget to
-   * update the SPI config file.
-   *
-   * In META-INF/services/org.apache.lucene.codecs.Codec we store
-   * the fully qualified codec name. If we rename the Codec class
-   * we have to update the SPI config file too. If we forget, this
-   * test reminds us by failing.
+   * This test ensures we registered our codec properly in the SPI file:
+   * META-INF/services/org.apache.lucene.codecs.Codec
    */
   @Test
   public void validateJavaServiceProviderConfigFile() throws Exception {
-    File serviceProviderConfigFile = findCodecServiceProviderFile();
-    Properties properties = new Properties();
-
-    try (InputStream inputStream = new FileInputStream(serviceProviderConfigFile)) {
-      properties.load(inputStream);
-    }
+    Codec codec = Codec.forName(RocanaSearchCodecV1.SHORT_NAME);
```

Comment:
Reading the JIRA, the conclusion is that `Codec.forName(String)` will traverse the classpath in the right way to properly include our custom `org.apache.lucene.codecs.Codec` file, and this test will always work regardless?